### DISCUSSION TITLE
Will Vue get all components like react and solid?

### DISCUSSION THREAD

**Discussion by Blinks44** [chakra-ui/ark#1183] [General]
Just curious, will Vue get Avatar, Switch (around a month in pull requests) and SegmentGroup, Datepicker, Presence and other planned components? Or Vue version has some problems in development?

Anyway you rock!

**Comment by cschroeter** [comment]
We plan to have all components across all supported frameworks eventually. Unfortunately the Vue contributions are not as frequent as for other frameworks. Happy to see a PR from you ;)
